BurnOutSharp/Tools/Utilities.cs

```diff
@@ -253,6 +254,44 @@ private static FileVersionInfo GetFileVersionInfo(string file)
             }
         }
 
+        /// <summary>
+        /// Get the SHA1 hash of a file, if possible
+        /// </summary>
+        /// <param name="path">Path to the file to be hashed</param>
+        /// <returns>SHA1 hash as a string on success, null on error</returns>
+        public static string GetFileSHA1(string path)
+        {
+            if (path == null)
+                return null;
+
+            try
+            {
+                SHA1Managed sha1 = new SHA1Managed();
```

Comment:
Please use the factory instead of creating a new managed instance directly.

```suggestion
                SHA1 sha1 = SHA1.Create();
```